### ISSUE TITLE
gameLoopFunction has wrong syntax parsing

### ISSUE THREAD

**Issue by Panossa** [42iscool42/MCC#29]
See image:
![snap_2017 07 21_17h17m23s_005_](https://user-images.githubusercontent.com/23323875/28470007-88a3c81c-6e38-11e7-8248-232fd7022a23.png)

Sample code:
```
gamerule gameLoopFunction something:something
gamerule gameLoopFunction something:temp/something

```

**Comment by Moesh** [comment]
Seems like it needs to be added in with our other fun specific parsing. I'll look into this if @42iscool42 don't get to it first.

**Comment by Moesh** [comment]
From #28 description by Panossa.

See image:
snap_2017 07 21_17h15m25s_004_
![image](https://user-images.githubusercontent.com/4852592/28487155-593d3fda-6e61-11e7-8e8f-1ed8e28ceb0d.png)


Sample code:

function something:something
advancement revoke @s only something:something

**Comment by 42iscool42** [comment]
This is another thing that is going to have to wait until the new version of the syntax highlighter is put in place.

**Comment by 42iscool42** [comment]
This is fixed in the beta 1.13 highlighting.
